packages/sdk/src/swap/assets.ts

```diff
@@ -31,13 +32,13 @@ const assetFactory =
     const assetAndChain = { asset, chain: assetChains[asset] } as AssetAndChain;
 
     return {
-      id: asset,
+      asset,
       chain: assetChains[asset],
       contractAddress: getTokenContractAddress(asset, network, false),
       decimals: assetDecimals[asset],
       name: assetName[asset],
       symbol: asset,
-      chainflipId: asset,
+      chainflipId: toCamelCase(asset),
```

Comment:
todo: revert this in https://github.com/chainflip-io/chainflip-sdk-monorepo/pull/550